src/styles/blocks/card/blockscard/card.scss

```diff
@@ -0,0 +1,44 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid $border-card;
+  border-radius: 5px;
+  transition: 300ms linear;
```

Comment:
It's a good practice to specify which properties should animate. This way, you can avoid unnecessary layout recalculations and improve rendering performance. For example, if you only want to animate the `transform` property, you should write `transition: transform 300ms linear;`